### ISSUE TITLE
Validation Loss and Metrics 

### ISSUE THREAD

**Issue by mdatres** [kkhoot/PAA#28]
Dear all, 

I just want to ask if it is possible to obtain a tensorboard with the training and validation losses plots. Furthermore, is it possible to perform an evaluation epoch after a certain number of training steps?

Thanks in advanced, 

mdatres

**Comment by kkhoot** [comment]
Hi, this repo doesn't support evaluation during training. I think you can use [this function](https://github.com/kkhoot/PAA/blob/master/tools/test_net.py#L31) instead. It runs eval for all existing checkpoints in the given directory, and automatically detects new checkpoints and runs eval for them. This will also plot AP scores on tensorboard.
Regarding tensorboard logs, I think you can add code in [here](https://github.com/kkhoot/PAA/blob/13d5721786a6aa3d3701d44a6d1216377c007a74/paa_core/engine/trainer.py#L38) to plot training losses.

**Comment by mdatres** [comment]
Thanks for the reply, 

mdatres
